docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,145 @@
+
+### Components Upgrade
+
+- VictoriaMetrics has been upgraded from 1.77.1 to 1.82.1.
+- As part of PMM 2.33.0, grafana has been updated to 9.2.5, which fixes some critical vulnerabilities. For more information, see [grafana 9.2.5](https://grafana.com/blog/2022/11/08/security-release-new-versions-of-grafana-with-critical-and-moderate-fixes-for-cve-2022-39328-cve-2022-39307-and-cve-2022-39306/)
+
+
+
+## New Features
+
+- [PMM-10889](https://jira.percona.com/browse/PMM-10889) - Starting with PMM2.33.0, while using the pmm-admin CLI command, you do not have to know the entire command. Just type the initial command and press Tab. The rest of the command will be autocompleted.
+
+    Depending on the shell you are using, add the following to your shells .rc file for autocomplete to work:
+
+    ```sh
+    source <(pmm-admin completion -c <your shell>)
+    E.g. source <(pmm-admin completion -c bash)
+    ```
+
+- [PMM-10955](https://jira.percona.com/browse/PMM-10955) - Timestamps for point-in-time recovery (PITR) restore. 
+
+    After restoring a PITR backup, you can now check the timestamp which the restore was performed to and the time when the restore was finalized. This information is available in the new **Finished at** column and **PITR timestamp** field in restore additional info under **Backup > Restores**. 
+
+
+
+
+## Improvements
+
+- [PMM-7000](https://jira.percona.com/browse/PMM-7000): [HA]: Staring with PMM 2.33.0, if the PMM Server goes down, metrics will be buffered on the monitored instance until PMM Server goes back online, which is when metrics will get pushed to the PMM Server. 
+
+
+- [PMM-10901](https://jira.percona.com/browse/PMM-10901): [Documentation]: To enable the users to use High Availability easily, we have added exclusive documentation for HA.
+
+- [PMM-8516](https://jira.percona.com/browse/PMM-8516): [Backup Management]:  When creating an a backup, the Vendor field was not automatically populated in the Back-up on demand dialog. This issue is now fixed. 
+
+- [PMM-10627](https://jira.percona.com/browse/PMM-10627): [DBaaS]: You can now create a single node cluster for PSMDB. 
+ 
+ - [PMM-10855](https://jira.percona.com/browse/PMM-10855): Avoid massive gRPC messages formatting for disabled log level
+
+
+- [PMM-10903](https://jira.percona.com/browse/PMM-10903): [DBaaS]: To simplify DBaas, you no longer have to manually enter the Public Address on the **PMM Settings > Advanced Settings** page. With DBaaS enabled, PMM will automatically detect and populate the **Public Address**.
+
+
+- [PMM-11085](https://jira.percona.com/browse/PMM-11085): [Backup Management]: We've temporarily removed the **Create backup** option under **All backups** page > **Actions** menu because this option can only create on-demand backups. You can continue to create both on-demand and scheduled backups using the **Create backup** button on this page. 
```

Comment:
Not sure if `temporarily` is a good word here. Probably we shouldn't promise it's not forever cuz we have plans to rework the UI heavily.

Reply:
@catalinaadam